c++/measures/g4_iw.cpp

```diff
@@ -52,16 +53,18 @@ namespace cthyb {
 
     // Allocate temporary NFFT two-frequency matrix M
     {
-      int resize_factor = 3; // How much bigger should the large mesh bee???
-      int nfreq         = resize_factor * std::max(n_fermionic, n_bosonic - 1 + n_fermionic);
+      // Determine maximum Matsubara index needed
+      int nfreq = std::max(3 * n_fermionic, n_bosonic + n_fermionic);
       gf_mesh<imfreq> iw_mesh_large{beta, Fermion, nfreq};
+      // In case of fermionic measurement, large mesh only needed for first component
```

Comment:
The m you set in the nfft_init is the truncation order of the window functions though. The sigma (oversampling factor) is not directly set by the interface as far as I know.

https://www-user.tu-chemnitz.de/~potts/nfft/guide3/doc/structnfft__plan.html